firestore/tests/unit/test__helpers.py

```diff
@@ -1216,6 +1217,9 @@ def _helper(self, option=None, do_transform=False, **write_kwargs):
         map_pb = document_pb2.MapValue(fields={
             'yum': _value_pb(bytes_value=value),
         })
+        if isinstance(option, ExistsOption):
+            write_kwargs = {'current_document' : {'exists': option._exists}}
```

Comment:
@chemelnucfin 

1. This should update ``write_kwargs`` but should not discard the previous values.
1. The value should be a protobuf, not a dictionary

Reply:
would an update work on write_args?

the value gets used in the protobuf below I think?